LN FAQ update (#1114)
btcpayserver/btcpayserver-doc: docs/FAQ/LightningNetwork.md

### I previously installed BTCPayServer without lightning, can I enable it?

BTCPay Server will allow you to set up your Lightning at any moment after creating your first store. 
We currently support three implementations of the Lightning Network. 

* [LND](https://github.com/lightningnetwork/lnd)
* [c-lightning](https://github.com/ElementsProject/lightning)
* [eclair](https://github.com/ACINQ/eclair)

![Connecting Lightning to store](../img/FAQ/btcpaylightningfaq1.jpg)

Want to learn more on how to set up your store with Lightning ? 

> Follow along in this [Guide](../LightningNetwork.md), and you will get your store connected to Lightning Network!

Reviewer: ```suggestion
Follow along the [Lightning Network](../LightningNetwork.md) guide, and you will get your store connected to Lightning Network!
```
